src/main/resources/config/liquibase/changelog/20231115144908_changelog.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="basak-akan" id="20231115133000-1">
+        <!-- Create lti_platform_configuration table -->
+        <createTable tableName="lti_platform_configuration">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="registration_id" type="varchar(255)"/>
+            <column name="client_id" type="varchar(255)"/>
+            <column name="issuer" type="varchar(255)"/>
+            <column name="authorization_uri" type="varchar(255)"/>
+            <column name="jkw_set_uri" type="varchar(255)"/>
+            <column name="token_uri" type="varchar(255)"/>
```

Comment:
Can you please verify that 255 is the proper length here?
Especially since there are URIs in there 255 might not be enough. And for the ids 255 might be more than we need.

Reply:
I just saw that these are the values that were used before, but it still would make sense to check if it makes sense to change these if you are already touching it.

Reply:
Optional: You could also check if some of these can be marked as "not null"

Reply:
Thank you for the feedback. I marked the necessary columns as not null. Also, I tried to verify the column lenghts below.
- registration_id is currently set as "artemis-" + UUID, however we are migrating old data from online_course_configuration into this table. For online_course_configuration table, registration_id was defined as course short name + UUID. Since I am not sure about all migrated course shortnames I would prefer to have varchar(255) for this column as before.
- client_id depends on the external LMS (i.e. Moodle) to how they set their client_id. I only verify one Moodle version, I cannot verify other tools or if it changes based on Moodle version.
- For url columns, I can verify that urls consist of domain name + corresponding endpoint. No other extra parameters. So varchar(255) should be enough.